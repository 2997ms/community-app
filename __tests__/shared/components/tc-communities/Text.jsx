import React from 'react';
import Rnd from 'react-test-renderer/shallow';
import Text from 'components/tc-communities/Text';

const rnd = new Rnd();

test('Snapshot match', () => {
  rnd.render((
    <Text>
      <div>content</div>
    </Text>
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();

  rnd.render((
    <Text
      theme={{
        container: 'container',
      }}
    >
      <div>content</div>
    </Text>
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();
});
