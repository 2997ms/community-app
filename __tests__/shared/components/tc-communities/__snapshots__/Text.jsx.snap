// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot match 1`] = `
<Text
  theme={
    Object {
      "container": "src-shared-components-tc-communities-Text-___style__container___GqwT0",
    }
  }
>
  <div>
    content
  </div>
</Text>
`;

exports[`Snapshot match 2`] = `
<Text
  theme={
    Object {
      "container": "src-shared-components-tc-communities-Text-___style__container___GqwT0 container",
    }
  }
>
  <div>
    content
  </div>
</Text>
`;
